api/Cargo.toml

```diff
@@ -17,11 +17,12 @@ serde_json = "1"
 slog = { version = "~2.3", features = ["max_level_trace", "release_max_level_trace"] }
 tokio = "0.1.7"
 tokio-core = "0.1.17"
-tokio-tls = "0.2"
-native-tls = "0.2"
```

Comment:
Does this mean you drop the openssl dependency? If so, can you update the Dockerfile?

Reply:
Thanks for spoting it, I checked Cargo.tomsl, it seems we don't use `openssl` in other places, will send a pr for dockerfile